src/EditorFeatures/VisualBasic/NavigationBar/VisualBasicNavigationBarItemService.vb

```diff
@@ -423,31 +426,31 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.NavigationBar
                 Dim method = TryCast(member, IMethodSymbol)
                 If method IsNot Nothing AndAlso method.PartialImplementationPart IsNot Nothing Then
                     method = method.PartialImplementationPart
-                    items.Add(New NavigationBarSymbolItem(
+                    items.Add(New SymbolItem(
                         method.ToDisplayString(displayFormat),
                         method.GetGlyph(),
                         spans,
-                        method.GetSymbolKey(),
-                        symbolIdIndexProvider.GetIndexForSymbolId(method.GetSymbolKey()),
+                        method.GetSymbolKey(cancellationToken),
+                        symbolIdIndexProvider.GetIndexForSymbolId(method.GetSymbolKey(cancellationToken)),
                         bolded:=spans.Count > 0,
                         grayed:=spans.Count = 0))
                 ElseIf method IsNot Nothing AndAlso IsUnimplementedPartial(method) Then
                     If workspaceSupportsDocumentChanges Then
-                        items.Add(New GenerateMethodItem(
+                        items.Add(New GenerateMethod(
                         member.ToDisplayString(displayFormat),
```

Comment:
Looks like indenting is screwed up here (you were fixing others so I'll assume this is fair game!)

Reply:
yes :)